tools/eval/datasets/otb100.py

```diff
@@ -0,0 +1,496 @@
+import os
+import numpy as np
+import cv2 as cv
+from colorama import Style, Fore
+from tqdm import tqdm
+from multiprocessing import Pool, cpu_count
+
+PRED_BBOXES_DICT = {}
```

Comment:
I suspect that the problem is this global variable. Please keep detection results in each corresponding class instead.